System.Net.TestData.nuspec

```diff
@@ -23,8 +23,20 @@
     <file src="System.Net.TestData\testclient1_at_contoso.com.cer" target="content\TestData\testclient1_at_contoso.com.cer" />
     <file src="System.Net.TestData\testclient1_at_contoso.com.p7b" target="content\TestData\testclient1_at_contoso.com.p7b" />
     <file src="System.Net.TestData\testclient1_at_contoso.com.pfx" target="content\TestData\testclient1_at_contoso.com.pfx" />
-	<file src="System.Net.TestData\testnoeku.contoso.com.cer" target="content\TestData\testnoeku.contoso.com.cer" />
-	<file src="System.Net.TestData\testnoeku.contoso.com.p7b" target="content\TestData\testnoeku.contoso.com.p7b" />
-	<file src="System.Net.TestData\testnoeku.contoso.com.pfx" target="content\TestData\testnoeku.contoso.com.pfx" />
+    <file src="System.Net.TestData\testclienteku.contoso.com.cer" target="content\TestData\testclienteku.contoso.com.cer" />
+    <file src="System.Net.TestData\testclienteku.contoso.com.p7b" target="content\TestData\testclienteku.contoso.com.p7b" />
+    <file src="System.Net.TestData\testclienteku.contoso.com.pfx" target="content\TestData\testclienteku.contoso.com.pfx" />
+    <file src="System.Net.TestData\testnoeku.contoso.com.cer" target="content\TestData\testnoeku.contoso.com.cer" />
+    <file src="System.Net.TestData\testnoeku.contoso.com.p7b" target="content\TestData\testnoeku.contoso.com.p7b" />
+    <file src="System.Net.TestData\testnoeku.contoso.com.pfx" target="content\TestData\testnoeku.contoso.com.pfx" />
+    <file src="System.Net.TestData\testselfsignedclienteku.contoso.com.cer" target="content\TestData\testselfsignedclienteku.contoso.com.cer" />
```

Comment:
testselfsignedclienteku.contoso.com.cer  is a binary file, the other .cer files are PEM encoded.  Is the right file in that filename? (It could be DER encoded, but since "one of these things is not like the other" I felt like mentioning it is good)